src/NzbDrone.Core/Indexers/HDBits/HDBitsSettings.cs

```diff
@@ -58,7 +58,8 @@ public enum HdBitsCodec
         H264 = 1,
         Mpeg2 = 2,
         Vc1 = 3,
-        Xvid = 4
+        Xvid = 4.
```

Comment:
shouldnt that be a trailing coma not a period. also squash and rebase your commits to keep history clean

Reply:
Yes, should be a `,`.

During a review squashing and rebasing should be left to the very end so individual commits can be compared. (We can squash and merge right from GitHub now).

Reply:
Did I do that wrong (squashing and rebasing)? I'm sorry, github can be complicated if not used frequently